app/service.js

```diff
@@ -20,31 +20,38 @@ function buildObj(pairs) {
   }, {});
 }
 
+function makeUrlAbsolute(base, relative) {
+  const relativeParsed = urlparse.parse(relative);
+
+  if (relativeParsed.host === null) {
+    return urlparse.resolve(base, relative);
+  }
+
+  return relative;
+}
+
 function getDocumentMetadata(url, window) {
   const doc = window.document;
   const metadata = getMetadata(doc);
 
-  metadata.url = url;
-  metadata.original_url = url;
-  metadata.provider_url = url;
-
-  metadata.favicon_url = metadata.icon_url;
-  if (!metadata.favicon_url) {
-    const parsedUrl = urlparse.parse(url);
-    metadata.favicon_url = `${parsedUrl.protocol}//${parsedUrl.host}/favicon.ico`;
-  }
-
-  metadata.image_url = metadata.image_url && metadata.image_url.replace(/^\/\//, 'https://');
-  metadata.images = [{
-    entropy: 1.0,
-    height: 500,
-    url: metadata.image_url,
-    width: 500,
-  }];
+  const responseData = {
+    url: url,
+    provider_url: url,
```

Comment:
Unrelated, but this logic is still weird and non-Embedly compliant.
Which reminds me, I was supposed to file a bug saying this logic is weird and non-Embedly compliant.